script.js

```diff
@@ -0,0 +1,17 @@
+const email = document.querySelector('#email');
+const senha = document.querySelector('#senha');
+const botaoLogin = document.getElementById('botaoLogin');
```

Comment:
Fala, Hiago! Olha, percebi que você utilizou o querySelector para as linhas 1 e 2. Que tal utilizar o querySelector para a linha 3 também? Lembre-se, o ideal é sempre termos um código padronizado, ok?

```suggestion
const email = document.querySelector('#email');
const senha = document.querySelector('#senha');
const botaoLogin = document.querySelector('#botaoLogin');
```